llvm/test/CodeGen/AMDGPU/llvm-intrinsics/reduce/add.ll

```diff
@@ -0,0 +1,3761 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
By main codegen directory, I mean llvm/test/CodeGen/AMDGPU/vector_reduce_<op>.ll, not this new subdirectory convention

Reply:
done